MODULE_IMPORT_REGRESSION_ANALYSIS.md

```
@@ -0,0 +1,260 @@
+
+```python
+# main.py
+from greetings import greeting  # greeting is a module, not a value
+```
+
+**Issue**: The bundler incorrectly treats the module `greeting` as a value and attempts to inline it.
+
+**Bundled Output Comparison**:
+
+- **Main branch**:
+  ```python
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)